python/pyspark/sql/functions.py

```diff
@@ -14489,13 +14476,16 @@ def call_function(funcName: str, *cols: "ColumnOrName") -> Column:
     |    2.0|
     +-------+
     >>> _ = spark.sql("CREATE FUNCTION custom_avg AS 'test.org.apache.spark.sql.MyDoubleAvg'")
+    ... # doctest: +SKIP
```

Comment:
This requires the test package has to be complied together, which we don't currently require.